.gitignore

```diff
@@ -1,2 +1,4 @@
 target/
 .idea
```

Comment:
Since you've got the '.*' rule you can get rid of the explicit '.idea' rule. Always best to keep things simple.

Reply:
Done 👍